components/menu/index.css

```diff
@@ -10,170 +10,676 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  }
+  
+  .spectrum-Menu-sectionHeading {
+    color: var(--highcontrast-menu-section-header-color, var(--mod-menu-section-header-color, var(--spectrum-menu-section-header-color)));
+  }
+
+  .spectrum-Menu-itemDescription {
+    color: var(--highcontrast-menu-item-description-color-down, var(--mod-menu-item-description-color-down, var(--spectrum-menu-item-description-color-down)));
+  }
+  
+  .spectrum-Menu-itemIcon:not(.spectrum-Menu-chevron, .spectrum-Menu-checkmark) {
+    fill: var(--highcontrast-menu-item-label-icon-color-down, var(--mod-menu-item-label-icon-color-down, var(--spectrum-menu-item-label-icon-color-down)));
+  }
+
+  .spectrum-Menu-itemValue {
+    color: var(--highcontrast-menu-item-value-color-down, var(--mod-menu-item-value-color-down, var(--spectrum-menu-item-value-color-down)));
+  }
+
+  .spectrum-Menu-chevron {
+    fill: var(--highcontrast-menu-collapsible-icon-color, var(--mod-menu-collapsible-icon-color, var(--spectrum-menu-collapsible-icon-color)));
+  }
 
-  &:focus {
-    outline: none;
+  .spectrum-Menu-checkmark {
+    fill: var(--highcontrast-menu-checkmark-icon-color-down, var(--mod-menu-checkmark-icon-color-down, var(--spectrum-menu-checkmark-icon-color-down)));
+  }
+}
+
+.spectrum-Menu-item.is-disabled,
+.spectrum-Menu-item[aria-disabled="true"] {
+  background-color: transparent;
+
+  .spectrum-Menu-itemLabel,
+  .spectrum-Menu-sectionHeading {
+    color: var(--highcontrast-menu-item-label-content-color-disabled, var(--mod-menu-item-label-content-color-disabled, var(--spectrum-menu-item-label-content-color-disabled)));
+  }
+  
+  .spectrum-Menu-itemDescription {
+    color: var(--highcontrast-menu-item-description-color-disabled, var(--mod-menu-item-description-color-disabled, var(--spectrum-menu-item-description-color-disabled)));
+  }
+  
+  .spectrum-Menu-itemIcon {
+    fill: var(--highcontrast-menu-item-label-icon-color-disabled, var(--mod-menu-item-label-icon-color-disabled, var(--spectrum-menu-item-label-icon-color-disabled)));
   }
 
-  &.is-selected {
-    /* Redundant, but included for backwards compatibility */
-    padding-inline-end: calc(
-      var(--spectrum-listitem-texticon-padding-right) -
-        var(--spectrum-popover-border-size)
-    );
+  &:hover {
+    cursor: default;
 
+    .spectrum-Menu-itemIcon {
+      fill: var(--highcontrast-menu-item-label-icon-color-disabled, var(--mod-menu-item-label-icon-color-disabled, var(--spectrum-menu-item-label-icon-color-disabled)));
+    }
+  }
+}
+
+.spectrum-Menu-itemIcon {
+  grid-area: iconArea;
+  align-self: center;
+}
+
+.spectrum-Menu-checkmark {
+  grid-area: checkmarkArea;
+}
+
+/* menus with "selectable" menuitems */
+.spectrum-Menu.is-selectable {
+  /* provide inline-start spacing for potential checkmarks */
+  .spectrum-Menu-item {
+    padding-inline-start: var(--spectrum-menu-item-selectable-edge-to-text-not-selected);
```

Comment:
Does this need a `--mod` override token?

Reply:
Yep, looks like I missed the `--mod`s. I'll add them now.